tests/Feature/SmartcarsTest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Tests\Feature;
```

Comment:
Can we move this into its own namespace? Perhaps call the test something like `SmartcarsInterfaceTest` as, at some stage, I'd like to test the API